packages/node/src/schematics/init/schema.json

```diff
@@ -4,6 +4,18 @@
   "title": "Add Nx Node Schematics",
   "type": "object",
   "properties": {
+    "unitTestRunner": {
+      "description": "Adds the specified unit test runner as well",
+      "type": "string",
+      "enum": ["jest", "none"],
+      "default": "jest"
+    },
+    "e2eTestRunner": {
+      "description": "Adds the specified unit test runner as well",
```

Comment:
As I see from the schema `node` schematic does not have e2e test runner, so maybe this should be removed